webpack.config.js

```diff
@@ -0,0 +1,33 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+  entry: './src/index.js',
```

Comment:
- Hey, great setting up the web-pack, however, we still need to set a mode in which we want the app to run, it could be `development` or `production` as this missing property is making your app throw an error. kindly fix this by adding a mode. e:g 
- Add the  `mode: 'development',` before the `entry: './src/index.js',`